docs/src/modules/java-protobuf/partials/actions.adoc

```diff
@@ -4,7 +4,9 @@ Actions are stateless functions that can be used to implement different uses cas
 
 - a pure function
 - request conversion - you can use Actions to convert incoming data into a different format before forwarding a call to a different component.
-- publish and subscribe to events 
```

Comment:
As a facade or controller to fan out multiple calls to different components?

Reply:
added